user_account/api/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+from user_account.api.serializers import RegistrationSerializer
+
+
+@api_view(
+    [
+        "POST",
+    ]
+)
```

Comment:
this decorator looks like a formatting oopsie, why not in a single line?

Reply:
It was black who did it, how can I tell him not to next time?